src/main/java/org/galatea/starter/entrypoint/BaseRestController.java

```diff
@@ -16,9 +18,21 @@
    * Adds the specified requestId to the context for this request (if not null).
    */
   protected void processRequestId(final String requestId) {
+    //This is a temporary solution for cleaning special characters from the REST request params
+    //It will not fix newlines in the parameters or JSON body being printed by the @Log annotation
+    //Proper fix requires version 2.10.0 or higher of the log4j dependencies
+    //See Issue #243 for more information.
     if (requestId != null) {
-      log.info("Request received with id: {}", requestId);
-      Tracer.setExternalRequestId(requestId);
+      String cleanedRequestId;
+      try {
+        cleanedRequestId = UriUtils.encode(requestId, "ISO-8859-1");
+        log.info("Request received.  Cleaned id: {}", cleanedRequestId);
+      } catch (UnsupportedEncodingException e) {
```

Comment:
have a look at: https://projectlombok.org/features/SneakyThrows for this case.  In particular have a look at the 2nd "common use case".

Reply:
Change made